src/Symfony/Component/Console/Tests/Helper/TableTest.php

```diff
@@ -550,6 +550,9 @@ public function testRowSeparator()
 TABLE;
 
         $this->assertEquals($expected, $this->getOutputContent($output));
+
+        // Test if fluent interface on addRow() with a single TableSeparator() works ok
```

Comment:
@Tobion  Should this be added as such a message instead of a comment. Somehow, I would at least like to tell why this assertion was added in the first place?